src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -168,7 +168,13 @@ class DropdownMenu extends React.Component<Props, State> {
             !menuEl.contains(event.target) &&
             !menuButtonEl.contains(event.target)
         ) {
+            const { onMenuClose } = this.props;
+
             this.closeMenu();
+
+            if (onMenuClose) {
```

Comment:
Could we just default onMenuClose to noop?

Reply:
@ConradJChan I just updated this file with your suggestion. Could you review again? Thanks!